test.slice

```diff
@@ -1,6 +1,7 @@
 
 // This is a file ending in `.slice`
 
+// Random comment.
 module Hello::There;
```

Comment:
And this file is correctly highlighted (by not being highlighted at all).
But, if you make a suggestion it's using the `.ice` highlighter when it shouldn't be:
```suggestion
module Hello::Theres {
```